read_test.go

```diff
@@ -0,0 +1,41 @@
+package pdf_test
+
+import (
+	"image/jpeg"
+	"testing"
+
+	"rsc.io/pdf"
+)
+
+func TestReaderExtractXObjectDCTDecode(t *testing.T) {
+	const (
+		testscan = "testdata/testscan.pdf"
```

Comment:
50K seems a bit large for such a basic test imho. It does not have to be a real scan coming from your scanner, as long as it is structured the same. Is it difficult to forge a pdf with a an  image of less than 1K in it?
Examples of tiny images: camlistore.org/pkg/images/testdata